tests/cases/fourslash/server/declarationMapsGoToDefinitionRelativeSourceRoot.ts

```diff
@@ -0,0 +1,40 @@
+/// <reference path="../fourslash.ts" />
+// @Filename: index.ts
+////export class Foo {
+////    member: string;
+////    /*2*/methodName(propName: SomeType): void {}
+////    otherMethod() {
+////        if (Math.random() > 0.5) {

```

Comment:
Could you try to cut the repro down to size?

Reply:
I mean, I didn't want to since it just a copy of the it her test with a relative sourceroot patched in. :3